src/components/App.js

```diff
@@ -102,11 +101,10 @@ class App extends Component {
         <SectionThree
           values={this.state}
         />
-        <SectionFour />
         {this.state.israel_results && this.state.new_zealand_results && <SectionFive
```

Comment:
why is it only checking for two countries here...?

Reply:
@Br3nda good spotting, fixing now